twitter/api.py

```diff
@@ -329,16 +339,19 @@ def __call__(self, **kwargs):
 
         req = urllib_request.Request(url_base, data=body, headers=headers)
         if self.retry:
-            return self._handle_response_with_retry(req, uri, arg_data, _timeout)
+            return self._handle_response_with_retry(req, uri, arg_data, _timeout, self.verify_context)
         else:
-            return self._handle_response(req, uri, arg_data, _timeout)
+            return self._handle_response(req, uri, arg_data, _timeout, self.verify_context)
 
-    def _handle_response(self, req, uri, arg_data, _timeout=None):
+    def _handle_response(self, req, uri, arg_data, _timeout=None, verify_context=False):
         kwargs = {}
         if _timeout:
             kwargs['timeout'] = _timeout
         try:
-            handle = urllib_request.urlopen(req, **kwargs)
+            context = None
+            if not verify_context:
+                context = ssl._create_unverified_context()
+            handle = urllib_request.urlopen(req, **kwargs, context=context)
```

Comment:
Hi and thanks for your contribution.
Although this breaks unit tests : context=context has to be before **kwargs, this is a syntax error: https://travis-ci.org/sixohsix/twitter/jobs/531693487